cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -534,7 +540,15 @@ private ApiClusterTemplate getCmTemplate(TemplatePreparationObject templatePrepa
 
     private void configureCmMgmtServices(TemplatePreparationObject templatePreparationObject, String sdxCrn, Telemetry telemetry,
             String sdxContextName, ProxyConfig proxyConfig) throws ApiException {
-        Optional<ApiHost> optionalCmHost = getCmHost(templatePreparationObject, apiClient);
+        Optional<ApiHost> optionalCmHost;
+        if (null != templatePreparationObject.getStackType()
+            && templatePreparationObject.getStackType().equals(StackType.DATALAKE)
+            && isVersionNewerOrEqualThanLimited(templatePreparationObject.getBlueprintView().getVersion(), CLOUDERA_STACK_VERSION_7_2_17)
+            && containsIgnoreCase(templatePreparationObject.getBlueprintView().getBlueprintText(), SDX_ENTERPRISE_DATALAKE_TEXT)) {
```

Comment:
isn't there any better solution for this? this seems fragile. Like if you change the bp description it would fail this check
Also if this stays, we have to ensure somehow that in future blueprints the same text is always present when introducing a new runtime version

Reply:
Ideally, we would like to have this logic for all the DL shapes with more than one host group.
Until we have a test coverage it would be better to limit it to the enterprise shape.

Reply:
then please provide a solution which ensures that new blueprints would have the same description. I mean if somebody adds 7.2.19, but changes the description we have to catch it during PR time, instead of figuring out later what went wrong

Reply:
As discussed in slack, removing the check on blueprint text will add an entitlement as this logic is needed for both MD and Enterprise shapes.

Reply:
I gave it a thought, I feel it would be better to limit changes only to Enterprise shape as the testing on MediumDuty shape will be limited going forward.

I made changes so that the description is as expected by adding unit tests.

Reply:
Addressed your comment.